Tutorial split (#145)
pretix/pretix-docs: docs/tutorial/continuous-operation/payment.md

# Payment

We do not just want to offer tickets in our shop; we also want to get paid for them.
That is what we will take care of in this part of the tutorial.
We are going to take the following steps:

 - enabling payment via credit card using [Stripe](payment.md#stripe)
 - enabling payment via [bank transfer](payment.md#bank-transfers)

>  - setting up [invoices](payment.md#setting-up-invoices) to send out automatically with every order

Reviewer: ```suggestion
 - setting up [invoices](payment.md#setting-up-invoices) to be sent automatically with every order
```

For our museum, we are planning to receive payments by credit card through the payment provider Stripe and by bank transfer.
The prerequisites for this are an active Stripe merchant account and a bank account.

pretix will automatically log payments coming in through most payment providers, including Stripe.
As soon as Stripe records a payment, pretix marks the corresponding order in our shop as paid.
Bank transfers are the exception to that rule because pretix would need to monitor payments arriving at our bank account.

On pretix Hosted, it is possible to set up a connection to our bank account.
On all versions of pretix, we can monitor payments via bank transfer either by manually approving them as they come in, or by regularly importing digital bank statements.

<br>

![Page titled 'Payment settings', on the 'Payment providers' tab. There is a list of payment providers: Bank transfer, Gift card, PayPal, SEPA debit and Stripe.](../../assets/screens/payment/settings.png "Payment settings screenshot")

In order to set up payment providers, we will navigate to our personal dashboard by clicking :btn-icon:i-pretix:pretix.eu: in the top left corner of the website.

> We will then select our event in the list of "Your upcoming events", open :btn-icon:fa3-wrench: Settings: in the sidebar and click the :btn:Payment: subentry.

Reviewer: > We will then select our event in the list of "Your upcoming events", open :btn-icon:fa3-wrench: Settings: in the sidebar and click the :btn:Payment: subentry.

Im Kontext dieses Tutorials ("Was für ein Event? Ich hab gar keine Veranstaltung, ich hab nur Öffnungszeiten") nach wie vor gar nicht so intuitiv.

This page displays a list of payment providers.
The list should include bank transfer and Stripe.
By default, all payment providers in this list except gift card will be disabled.

!!! Note
    If this list does not display the payment providers you want to use, that means the corresponding plugin is not enabled.
    In order to activate such a plugin, navigate to :navpath:Your event → Settings → Plugins: and open the :btn:Payment providers: tab.
    Click the :btn:Enable: button next to the plugin you want to use.

>     You can tell that a plugin has is active by green ":fa3-check: Active" tag.

Reviewer: ```suggestion
    You can tell that a plugin is active by the green ":fa3-check: Active" tag.
```

We will also check the box next to "Enable payment method".
Then, we will click the :btn:Save: button and return to the payment settings.
Bank transfer, gift card, and Stripe should all have a green ":fa3-check: Enabled" tag next to them now.

!!! Note
    By default, the pretix software cannot to monitor payments arriving at your bank account.
    On pretix Hosted, you can set up automated bank imports.
    On pretix Community and pretix Enterprise, you have to notify the pretix software of incoming payments—either by manually approving payments as complete, or by regularly importing digital bank statements.
    Read our guide on [monitoring incoming payments](../../guides/payment/bank-transfer.md#monitoring-incoming-payments-automatically) before using bank transfer as a payment method in your shop.

For more information on using bank transfers, see [bank transfers](../../guides/payment/bank-transfer.md).

## Setting up invoices

![Page titled 'Invoice settings', displaying options for generating invoices, which sales channels to generate them for, and attaching them to emails.](../../assets/screens/payment/invoice.png "Invoice settings screenshot")

> We will navigate to the invoicing settings by going to our event, opening :btn-icon:fa3-wrench: Settings: in the sidebar and clicking the :btn:Invoicing: subentry.

Reviewer: > We will navigate to the invoicing settings by going to our event, opening :btn-icon:fa3-wrench: Settings: in the sidebar and clicking the :btn:Invoicing: subentry.

Könnte als Tutorial vielleicht noch einen Satz zum "warum" verkraften.

On the :btn:Invoice generation: tab, we will set the "Generate invoices" option to "Automatically after payment or when required by payment method."

> We will then switch to the :btn:Issuer details: tab and provide the name and full address of our organization.

Reviewer: > We will then switch to the :btn:Issuer details: tab and provide the name and full address of our organization.

Organization?

pretix will use the data we enter here for all invoices we generate from this point on.

If we now click the :btn:Save and show preview: button, our browser is going to download a PDF preview of the invoice.

> This invoice will include our organization's address, the invoice number TUTCON27-PREVIEW and an example listing of purchases with a partial payment.

Reviewer: ```suggestion
This invoice will include our organization's address, the invoice number TUTMUS27-PREVIEW and an example listing of purchases with a partial payment.
```
etc.
